sources/stripe_analytics_pipeline.py

```diff
@@ -73,22 +73,22 @@ def load_incremental_endpoints(
     load_info = pipeline.run(source)
     print(load_info)
 
-    # load nothing, because incremental loading and end date limit
-    source = incremental_stripe_source(
-        endpoints=endpoints,
-        initial_start_date=initial_start_date,
-        end_date=end_date,
-    )
-    load_info = pipeline.run(source)
-    print(load_info)
-
-    # load only the new data that created after end_date
-    source = incremental_stripe_source(
-        endpoints=endpoints,
-        initial_start_date=initial_start_date,
-    )
-    load_info = pipeline.run(source)
-    print(load_info)
+    # # load nothing, because incremental loading and end date limit
```

Comment:
Did they become flaky should we mark them as flaky if so?

Reply:
no it works good, it just an example how incremental loading works. I commented it because it will affect tuba project